### PR TITLE
Update application list and cases view with startdates, sorting, filtering

### PR DESCRIPTION
**Pull request by svenow**
Contains the following Views adjustments in each view:

- a new field _start_date_ has been added
- a couple of fields can now be ordered (status, title, identifier, titles, dates)
- field order has been changed (identifiers, titles, start_date, status)

Only in _civiremote_funding_application_list_ view:

- a new filter has been added: _title_ (process)

systopia-reference: 24749

### CODE REVIEW

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 1]
```diff
@@ -361,6 +361,58 @@ display:
...
           separator: ', '
+        application_process_start_date:
+          id: application_process_start_date
+          table: cmrf_views_civiremote_funding_caseinfo
+          field: application_process_start_date
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: cmrf_views_date
+          label: 'Beginn der Maßnahme'
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_application_list.yml:372]: Labels should be in English and translated via translation file.

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 2]
```diff
@@ -361,6 +361,58 @@ display:
...
+          element_label_class: ''
+          element_label_colon: true
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          date_format: custom
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_application_list.yml:413]: There should be no hard-coded date format. I'd suggest to use `short` as it is the case for other date fields.

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 3]
```diff
@@ -1052,6 +1104,52 @@ display:
...
+          group_type: group
+          admin_label: ''
+          plugin_id: cmrf_views_filter_text
+          operator: contains
+          value: ''
+          group: 1
+          exposed: true
+          expose:
+            operator_id: application_process_title_op
+            label: Titel
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_application_list.yml:1121]: Labels should be in English and translated via translation file.

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml [hunk 4]
```diff
@@ -306,6 +306,58 @@ display:
...
+          element_label_class: ''
+          element_label_colon: true
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          date_format: custom
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_combined_application_process_list.yml:358]: There should be no hard-coded date format. I'd suggest to use `short` as it is the case for other date fields.

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml [hunk 5]
```diff
@@ -306,6 +306,58 @@ display:
...
+          custom_date_format: d.m.o
+          timezone: Europe/Berlin
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_combined_application_process_list.yml:360]: A timezone should be not part of the view.

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml [hunk 6]
```diff
@@ -306,6 +306,58 @@ display:
...
           separator: ', '
+        start_date:
+          id: start_date
+          table: cmrf_views_civiremote_funding_application_process
+          field: start_date
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: cmrf_views_date
+          label: 'Beginn der Maßnahme'
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_combined_application_process_list.yml:317]: Labels should be in English and translated via translation file.

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 7]
```diff
@@ -1089,7 +1187,15 @@ display:
             required: false
             remember: false
             multiple: false
-            remember_roles: null
+            remember_roles:
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_application_list.yml:1190]: Should be set to `null`.

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 8]
```diff
@@ -1052,6 +1104,52 @@ display:
...
+            description: ''
+            use_operator: false
+            operator: application_process_title_op
+            operator_limit_selection: false
+            operator_list: {  }
+            identifier: application_process_title
+            required: false
+            remember: false
+            multiple: false
+            remember_roles:
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_application_list.yml:1131]: Should be set to `null`.

**Review by dontub — CHANGES_REQUESTED** [review]
Can you also fix the PR title?

I guess "excel export has been merged from main" is no adjustment of this PR ;-)

File: config/optional/views.view.civiremote_funding_application_list.yml [hunk 9]
```diff
@@ -1137,15 +1231,15 @@ display:
...
+              sortable: true
               default_sort_order: asc
               align: ''
               separator: ''
               empty_column: false
               responsive: ''
             funding_case_id:
               sortable: true
-              default_sort_order: desc
+              default_sort_order: asc
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_application_list.yml:1242]: Is there a reason to use `asc`? I think by default the more recent applications should be at the top.

File: config/optional/views.view.civiremote_funding_combined_application_process_list.yml [hunk 10]
```diff
@@ -306,6 +306,58 @@ display:
...
           separator: ', '
+        start_date:
+          id: start_date
+          table: cmrf_views_civiremote_funding_application_process
+          field: start_date
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: cmrf_views_date
+          label: 'Start Date Activity'
```
Comment (dontub) [inline config/optional/views.view.civiremote_funding_combined_application_process_list.yml:317]: I think this should be `Activity Start Date`.

**Review by dontub — CHANGES_REQUESTED** [review]

File: config/optional/views.view.civiremote_funding_application_list.yml (hunk 9 above)
Comment (svenow) [reply config/optional/views.view.civiremote_funding_application_list.yml]: switched back to desc

**Review by dontub — APPROVED** [review]

**Comment by svenow** [comment]
- rebased from main
- removed dublicate elements from views

**Pull request merged**